### ISSUE TITLE
Implement autosizing of ChillerPerformance:Electric:EIR 'Reference Cooling Mode Evaporator Capacity'

### ISSUE THREAD

**Issue by jmarrec** [NREL/EnergyPlus#6445]
Issue overview
--------------

I'm testing a new object in OpenStudio, the `CentralHeatPumpSystem`. I have implemented a model with a central heat pump system and three plant loops. The simulation completes correctly. But the results are weird, I have never ran into this issue: I have NaNs all over the SQL / HTML output.

![image](https://user-images.githubusercontent.com/5479063/35438670-b857a472-0297-11e8-98d1-41e795154451.png)

It seems that the end use that's creating the problem is the Pumps one:

![image](https://user-images.githubusercontent.com/5479063/35438770-266eeaa6-0298-11e8-92f7-db1a68dcc15d.png)

Here's the guilty IDF:

[in.idf.txt](https://github.com/NREL/EnergyPlus/files/1667603/in.idf.txt)

The weather file used is USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw

I'm including the OSM for completeness, but you can't use it unless you build with the open PR https://github.com/NREL/OpenStudio/pull/2976.

[in.osm.txt](https://github.com/NREL/EnergyPlus/files/1667604/in.osm.txt)

### Details
Some additional details for this issue (if relevant):
 - Platform: Ubuntu 16.04
 - Version of EnergyPlus: Version 8.8.0-7c3bbe4830

### Checklist
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [ ] Defect file added: https://github.com/NREL/EnergyPlusDevSupport/tree/master/DefectFiles/6000s/6445
 - [ ] Ticket added to Pivotal for defect (development team task)
 - [ ] Pull request created (the pull request will have additional tasks related to reviewing changes that fix this defect)

**Comment by jmarrec** [comment]
I requested "Pump Electric Power" and "Pump Mass Flow Rate" at Detailed frequency for all pumps in my model. I have 3 identical Pump:VariableSpeed:

* CHW LOOP VSD PUMP 
* CNDW LOOP VSD PUMP
* HW LOOP VSD PUMP 

Looking at the sql file, CHW and CNDW are fine, I have values for both variables.
HW LOOP VSD PUMP on the other hand has values for pump mass flow rate, but "NULL" for the Pump Electric Power.

```
SELECT EnvironmentPeriodIndex, Month,Day,Hour,Minute,
    ReportingFrequency, KeyValue, Name, Units,
    Value
FROM ReportVariableWithTime
  WHERE UPPER(ReportingFrequency) IN ('HVAC SYSTEM TIMESTEP')
```

![image](https://user-images.githubusercontent.com/5479063/35439298-7085dd50-029a-11e8-9291-4f50140e167a.png)


I then requested running for sizing periods (one heating DD, one cooling design day).

Heating: 96 NaN
Cooling: 74 values at 0.0, 53 NaNs.

**Comment by jmarrec** [comment]
Running it after transition to V8.9 with current develop built in debug mode produces a hard crash due to division by zero on this line: https://github.com/jmarrec/EnergyPlus/blob/develop/src/EnergyPlus/PlantCentralGSHP.cc#L2355

On this line: https://github.com/jmarrec/EnergyPlus/blob/develop/src/EnergyPlus/PlantCentralGSHP.cc#L1085

(lldb) p ChillerHeater( ChillerHeaterNum ).RefCapClgHtg
(Real64) $0 = 0
(lldb) p ChillerHeater( ChillerHeaterNum ).RefPowerClgHtg
(Real64) $1 = -91999.079999999987

A few lines before you'll see there's a check for RefCapClgHtg is autosized or not, but there's no call to autosize it.... In my case it is autosized... So that Cop becomes 0 (and it wasn't, wouldn't it be a negative COP given that RefPowerClgHtg is negative...?)

**Comment by jmarrec** [comment]
Confirmed that if I hardside the ChillerPerformance:Electric:EIR [Reference Cooling Mode Evaporator Capacity](https://bigladdersoftware.com/epx/docs/8-8/input-output-reference/group-plant-equipment.html#field-reference-cooling-mode-evaporator-capacity), I no longer get the crash.

And if I run it in 8.8.0, I no longer have NaNs either

**Comment by Myoldmopar** [comment]
Closed via #7091
